src/backend/storeManagers/gog/games.ts

```diff
@@ -1140,6 +1152,17 @@ export async function update(
     (isWindows || existsSync(gameSettings.winePrefix))
   ) {
     await setup(appName, gameObject, false)
+  } else if (gameObject.platform === 'linux') {
+    const installer = join(gameObject.install_path, 'support/postinst.sh')
+    if (existsSync(installer)) {
+      logInfo(`Running ${installer}`, LogPrefix.Gog)
+      await spawnAsync(installer, []).catch((err) =>
```

Comment:
Are we sure these scripts always have their executable bit set?

Reply:
pretty sure, yeah, they are automatically added to the zip and include the flag. I guess we could add a safeguard to always set it.

Reply:
I see. Let's add that if we run into any issues, no point in adding safeguards where they aren't necessary